npm/pkg/dataplane/debug/trafficanalyzer.go

```diff
@@ -206,46 +164,47 @@ func generateTuple(src, dst *controllersv1.NpmPod, rule *pb.RuleResponse) *Tuple
 }
 
 func getHitRules(
-	src, dst *controllersv1.NpmPod,
+	src, dst *common.NpmPod,
 	rules []*pb.RuleResponse,
-	npmCache *controllersv1.Cache,
+	npmCache common.Cache,
 ) ([]*pb.RuleResponse, error) {
 
 	res := make([]*pb.RuleResponse, 0)
+
 	for _, rule := range rules {
-		matched := true
+		matched := false
```

Comment:
In V2 we have cut down a single rule, into a half match in top level ingress or egress chain and then jump to Netpol specific ingress or egress chain and match the 2nd half of the packet in a given direction, that change is still missing